clients/java/src/test/java/io/zeebe/client/ZeebeClientTest.java

```diff
@@ -47,4 +52,19 @@ public void shouldHaveDefaultValues() {
       assertThat(configuration.getDefaultRequestTimeout()).isEqualTo(Duration.ofSeconds(20));
     }
   }
+
+  @Test
+  public void shouldFailIfCertificateDoesNotExist() {
+    // given
+    thrown.expectCause(IsInstanceOf.instanceOf(FileNotFoundException.class));
+
+    // when
+    ZeebeClient.newClientBuilder().useSecureConnection().caCertificatePath("/wrong/path").build();
```

Comment:
You can in general still use the normal AssertJ stuff, i.e. 

```java
assertThatThrownBy(() -> 
  ZeebeClient.newClientBuilder()
    .useSecureConnection()
    .caCertificatePath("/wrong/path")
    .build()
).isInstanceOf(FileNotFoundException.class);
```

Which reads a little better imo and means we don't need the `Hamcrest` dependency here